ui/test/nuxeo-date-picker.test.js

```diff
@@ -32,6 +36,17 @@ function testValue(element, value, isUTC) {
   // expect(element._pickerValue).to.be.equal(local);
 }
 
+function testValueWithLocale(element, value, locale, isUTC) {
+  moment.locale(locale);
```

Comment:
Won't this change the locale for all usages of moment from this test on?

Reply:
😱 You're totally right, I should reset the locale after the test. I'll update the code. Thank you.